discovery-provider/src/models/models.py

```diff
@@ -178,8 +178,11 @@ def __repr__(self):
 class User(Base):
     __tablename__ = "users"
 
-    blockhash = Column(String, ForeignKey("blocks.blockhash"), nullable=False)
-    blocknumber = Column(Integer, ForeignKey("blocks.number"), nullable=False)
+    blockhash = Column(String, ForeignKey("blocks.blockhash"), nullable=True)
+    blocknumber = Column(Integer, ForeignKey("blocks.number"), nullable=True)
```

Comment:
so you wanna keep these fkeys to blocks and keep that around? i guess there's no harm in keeping it since we have removed the not null constraint

Reply:
I think we should keep it 1-1 with what's in the db